lib/releases.sh

```diff
@@ -47,3 +47,7 @@ fi
 if [[ "$CAPM3RELEASE" == "" ]]; then
   command -v jq &> /dev/null && echo "Failed to fetch CAPM3 release from Github" && exit 1
 fi
+
+# remove the next two lines
```

Comment:
Remove... when?

Reply:
They will be removed. Added here temporarily to bypass the following problem.
**The ansible scripts are overriding variables defined in bash scripts.**
They will be removed before merging this PR.